x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -177,11 +179,11 @@ type EventDisplayName = typeof displayNameRecord[keyof typeof displayNameRecord]
   typeof unknownEventTypeMessage;
 
 /**
- * Take a gross `schemaName` and return a beautiful translated one.
```

Comment:
:information_source: No fun allowed :straight_ruler:

Reply:
Just trying to get clarity.